circle.yml

```diff
@@ -31,7 +31,6 @@ commands:
                       if [ << parameters.target >> == base ]; then DOCKER_TAG=cypress/base:${BASE_IMAGE_TAG}; fi
                       if [ << parameters.target >> == browsers ]; then DOCKER_TAG=cypress/browsers:${BROWSERS_IMAGE_TAG}; fi
                       if [ << parameters.target >> == included ]; then DOCKER_TAG=cypress/included:${INCLUDED_IMAGE_TAG}; fi
-                      if [ << parameters.target >> == included-short-tag ]; then DOCKER_TAG=cypress/included:${INCLUDED_IMAGE_SHORT_TAG}; fi
```

Comment:
release a new tag whenever the full version tag updates, this saves a second build step,

latest will work the same.